config/initializers/json.rb

```diff
@@ -0,0 +1,24 @@
+require 'json/version'
+module JSON
```

Comment:
should we check the json version and add it only for the specific version that breaks?